### PR TITLE
Remote updates with ansible-pull

### PR DESCRIPTION
**Pull request by jasoncorlett**
**Changes:**
* Paths in ansible/nimbuserver.yml now relative, allows updating with ansible-pull
* New nimbus-server-update script to run ansible-pull easily
* Tweaks to nimbusserver.json so ansible will work with relative paths, and make using a new CentOS iso image easier

**To test:**
```
ansible-pull -U git://github.com/jasoncorlett/nimbusserver-ansible -C remote-update ansible/nimbusserver.yml
```
OR, if the script is installed:
```
export NIMBUS_UPDATE_URL="git://github.com/jasoncorlett/nimbusserver-ansible"
export NIMBUS_UPDATE_BRANCH="remote-update"
nimbus-server-update
```

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,6 +1,14 @@
 # nimbusserver-ansible
 All notable changes to this project will be documented in this file. The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
+# [2019.04] - 2019-04-09
```
Comment (jhrabi) [inline changelog.md]: I'm looking at making the naming convention Year+R(number). This way it doesn't matter when we release it,
I'm going to confirm this with Keith.

**Review by jhrabi — COMMENTED** [review]
We need to update quickstart.html to add the new version as well.

**Review by jhrabi — APPROVED** [review]
LGTM

**Review by jhrabi — APPROVED** [review]
LGTM

**Review by jhrabi — APPROVED** [review]
LGTM

File: ansible/nimbusserver.yml [hunk 2]
```diff
@@ -221,6 +221,10 @@
       regexp: '^(.*)MaxSessions(.*)$'
       replace: 'MaxSessions=1'
       backup: yes
+
+  - name: Remove existing chrome directory
```
Comment (jhrabi) [inline ansible/nimbusserver.yml]: Isn't this redundant with the change made below for the bookmarks?

Comment (jasoncorlett) [reply ansible/nimbusserver.yml]: Indeed, forgot I already added this. I'll remove this one with my next push.

**Review by jhrabi — APPROVED** [review]
LGTM

**Pull request merged**